### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by doubldragon**
adding lease_time to subnets table

### CODE REVIEW

**Comment by jctu225** [comment]
+1

File: app/Http/Controllers/EquipmentTypeController.php [hunk 1]
```diff
@@ -16,9 +16,9 @@ public function index()
     {
         //
         $equipment_type = EquipmentType::all()
-            ->pluck('name')
+            ->pluck('id', 'name')
             ->toArray();
-        return $equipment_type;
+        var_dump($equipment_type);
```
Comment (stockdalemark1978) [inline app/Http/Controllers/EquipmentTypeController.php]: Did you mean to leave this as a var_dump?

Comment (doubldragon) [reply app/Http/Controllers/EquipmentTypeController.php]: This was from daebrion's function, and I was tinkering with it but changed it back to leave it how it was.

Comment (dhopwood) [reply app/Http/Controllers/EquipmentTypeController.php]: the var dump controller

**Comment by dhopwood** [comment]
+1

**Pull request merged**